airbyte-integrations/connectors/destination-s3/src/main/resources/spec.json

```diff
@@ -167,6 +167,13 @@
                 "default": "No flattening",
                 "enum": ["No flattening", "Root level flattening"]
               },
+              "stringify": {
```

Comment:
I would rather not have a new spec for S3, especially given the work that's going on s3-v2